eduNEXT/drydock
# feat: add templates with tutor14 support

### Description
This PR adds support for tutor 14+ k8s templates and needed configurations. Roughly, this is what changed:

### What's changed

1. Containers entrypoint for cms-worker and lms-worker from:

```
["celery", "worker", "--app=cms.celery", "--loglevel=info", "--hostname=edx.cms.core.default.%%h", "--maxtasksperchild", "100", "--exclude-queues=edx.lms.core.default"]
["celery", "worker", "--app=lms.celery", "--loglevel=info", "--hostname=edx.lms.core.default.%%h", "--maxtasksperchild=100", "--exclude-queues=edx.cms.core.default"]
```

To:
```
["celery", "--app=cms.celery", "worker", "--loglevel=info", "--hostname=edx.cms.core.default.%%h", "--max-tasks-per-child", "100", "--exclude-queues=edx.lms.core.default"]
["celery", "--app=lms.celery", "worker", "--loglevel=info", "--hostname=edx.lms.core.default.%%h", "--max-tasks-per-child=100", "--exclude-queues=edx.cms.core.default"]
```
2. `apps/openedx/config/partials/auth.yml` doesn't have `CONTENTSTORE: null`
3. mongob parameters changed replicaSet to None
    
    ```python
    mongodb_parameters = {
         "password": None,
         {% endif %}
         "db": "{{ MONGODB_DATABASE }}",
    +    "replicaSet": None,
     }
    ```
    
4. It was added to `openedx/config.settings/partials/common_lms.py`:
    
    ```python
    +BULK_EMAIL_SEND_USING_EDX_ACE = True
    +
    +# Make it possible to hide courses by default from the studio
    +SEARCH_SKIP_SHOW_IN_CATALOG_FILTERING = False
    ```

### How to test
1. Use this branch MJG/tutor-14
2. Use the `manifests-a-j-goddard` branch `MJG/nutmeg-installation` (I know it's nuez)
3. Generate manifests using the reference. Notice we're using `template_root: "tutor14"`
